config/200-clusterrole.yaml

```diff
@@ -53,4 +53,41 @@ metadata:
 rules:
   - apiGroups: [""]
     resources: ["secrets"]
+    verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tekton-triggers-eventlistener
```

Comment:
what do you think of calling this simply `eventlistener-roles` and the other one `eventlistener-clusterroles`

Reply:
end users might have many things installed on the cluster so if we keep `tekton-triggers-` as prefix so wouldn't it be easier to understand where the clusterrole came from.

Reply:
and also it would be uniform with other resources we package

Reply:
fair enough!